test/test_proxy_tensor.py

```diff
@@ -1596,23 +1597,23 @@ def f(args, kwargs, extra_args, extra_kwargs):
         self.assertEqual(new_out, old_out)
 
 class TestProxyTensorOpInfo(TestCase):
-    @ops(op_db, allowed_dtypes=(torch.float,))
+    @ops(op_db + custom_op_db, allowed_dtypes=(torch.float,))
```

Comment:
Can you paste the generated graph from make_fx here? Just wanted to see how the end graph looks like.

Reply:
```
import torch
from torch.testing._internal.custom_op_db import numpy_mul
from functorch import make_fx

def f(x, y):
    return numpy_mul(x, y)


x = torch.randn(3)
y = torch.randn(2, 1)

gm = make_fx(f, tracing_mode='symbolic')(x, y)
print(gm.code)
```
produces:
```
def forward(self, x_1, y_1):
    numpy_mul = torch.ops._torch_testing.numpy_mul.default(x_1, y_1);  x_1 = y_1 = None
    return numpy_mul
```